### PR TITLE
Feat/templating

### PR DESCRIPTION
**Pull request by yevgenypats**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -68,133 +45,40 @@ const run = async () => {
...
-				if (existingPr) await git.removePrWarning()
-
 				return
 			}
 
-			// If there are still local changes left (i.e. not committed each file separately), commit them before pushing
-			if (hasChanges === true) {
-				core.debug(`Creating commit for remaining files`)
+			core.info(`Creating commit for remaining files`)
+			core.info(await git.commit("chore: Sync form .github"))
```
Comment (bbernays) [inline src/index.js:76]:
```suggestion
			core.info(await git.commit("chore: Sync from .github"))
```

File: src/index.js [hunk 2]
```diff
@@ -68,133 +45,40 @@ const run = async () => {
...
-					core.info(`Adding reviewer(s) "${ REVIEWERS.join(', ') }" to PR`)
-					await git.addPrReviewers(REVIEWERS)
-				}
-
-				if (TEAM_REVIEWERS !== undefined && TEAM_REVIEWERS.length > 0 && !FORK) {
-					core.info(`Adding team reviewer(s) "${ TEAM_REVIEWERS.join(', ') }" to PR`)
-					await git.addPrTeamReviewers(TEAM_REVIEWERS)
-				}
-			}
+			const pullRequest = await git.createOrUpdatePr("chore: Sync form .github")
```
Comment (bbernays) [inline src/index.js:79]:
```suggestion
			const pullRequest = await git.createOrUpdatePr("chore: Sync from .github")
```

**Comment by erezrokah** [comment]
Followed up on this in https://github.com/BetaHuhn/repo-file-sync-action/pull/201

**Comment by erezrokah** [comment]
Followed up on this in https://github.com/BetaHuhn/repo-file-sync-action/pull/201

**Comment by erezrokah** [comment]
Closing as stale

**Pull request closed without merging**